website/docs/artist_hosts_houdini.md

```diff
@@ -13,21 +13,77 @@ sidebar_label: Houdini
 - [Publish](artist_tools_publisher)
 - [Library Loader](artist_tools_library_loader)
 
-## Publishing Alembic Cameras
+## Ayon Houdini Publish Process
+:::note
+Ayon shouldn't be opinionated on your workflow, it only ensures that your work meets your studio's technical specifications.
```

Comment:
These are Artist docs. What does it mean that it **shouldn't be opinionated**? Should they be doing something so it remains not being opiniated?

If you mean to say that Ayon is not opiniated it might be better worded as:

```suggestion
Ayon tries to not be opinionated on your workflow, it only ensures that your work meets your studio's technical specifications using studio project settings customizations to customize it for particular studio or project's needs.
```

Reply:
Thanks for help. 
I added this because some artists may have concerns that they are forced to do things a certain way